src/App.tsx

```diff
@@ -8,8 +8,24 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { useAppContext } from './context/store';
+import { Notification } from './components/Notification';
+import { Error } from './types/Notification';
 
 export const App: React.FC = () => {
+  const {
+    state: { selectedUser, selectedPost, loading, error },
+    methods: { setLoading, setPosts, setSelectedPost },
+  } = useAppContext();
+
+  useEffect(() => {
+    if (selectedUser) {
+      setLoading(true);
+      setSelectedPost(null);
+      setPosts(selectedUser.id).finally(() => setLoading(false));
+    }
+  }, [selectedUser]);
```

Comment:
Fix it 
<img width="1184" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-posts/assets/64750958/2a7ff180-75f4-4dca-95f5-2f6039a6638b">

Reply:
This VSCode suggestion breaks the code.